### PR TITLE
Add support for VK_NV_low_latency and VK_NV_low_latency2

### PR DESCRIPTION
**Pull request by davidlunarg**
Also removed some debug output in generated code.

### CODE REVIEW

**Review by locke-lunarg — APPROVED** [review]

**Comment by locke-lunarg** [comment]
I'm ok to remove `body.append(makeGen('// {arg.name}', locals(), 0))` to remove some debug output. But that's @MarkY-LunarG  added. I wonder if @MarkY-LunarG  agree on it?

File: framework/generated/vulkan_generators/vulkan_cpp_consumer_body_generator.py [hunk 1]
```diff
@@ -795,8 +795,6 @@ def makeConsumerFuncBody(self, returnType, name, values):
             if arg.base_type in self.handle_names:
                 handleObjectType = makeObjectType(arg.base_type)
 
-            body += makeGen('// {arg.name}', locals(), indent=0)
```
Comment (locke-lunarg) [inline framework/generated/vulkan_generators/vulkan_cpp_consumer_body_generator.py:798]: I'm ok to remove this code to remove some debug output. But the code was added by @MarkY-LunarG. I wonder if @MarkY-LunarG agree on it?

Comment (bradgrantham-lunarg) [reply framework/generated/vulkan_generators/vulkan_cpp_consumer_body_generator.py:798]: I think removing it sounds right.  @MarkY-LunarG

Comment (MarkY-LunarG) [reply framework/generated/vulkan_generators/vulkan_cpp_consumer_body_generator.py:798]: I'm okay with removing this.

**Pull request merged**